rules/windows/process_creation/proc_creation_win_susp_psexex_paexec_escalate_system.yml

```diff
@@ -1,27 +1,67 @@
 title: PsExec/PAExec Escalation to LOCAL SYSTEM
 id: 8834e2f7-6b4b-4f09-8906-d2276470ee23
+related:
+    - id: 207b0396-3689-42d9-8399-4222658efc99
+      type: similar
 status: experimental
 description: Detects suspicious flags used by PsExec and PAExec to escalate a command line to LOCAL_SYSTEM rights
 references:
     - https://docs.microsoft.com/en-us/sysinternals/downloads/psexec
     - https://www.poweradmin.com/paexec/
     - https://www.fireeye.com/blog/threat-research/2020/10/kegtap-and-singlemalt-with-a-ransomware-chaser.html
-author: Florian Roth
+author: Florian Roth, Nasreddine Bencherchali
 date: 2021/11/23
-modified: 2022/08/24
+modified: 2022/10/26
 logsource:
     category: process_creation
     product: windows
 detection:
-    selection: # Escalation to LOCAL_SYSTEM
-        CommandLine|endswith: ' -s cmd.exe'
+    selection_sys: # Escalation to LOCAL_SYSTEM
```

Comment:
first idea
```yaml
    # Escalation to LOCAL_SYSTEM
    selection_sys: 
        CommandLine|contains:
            # The "-" can also be replaced with "/"
            # The order of args isn't important
            - ' -s '
            - ' /s '
            - ' -s -i '
            - ' /s /i '
            - ' /s -i '
            - ' -s /i '
            - ' -i -s '
            - ' /i /s '
            - ' -i /s '
            - ' /i -s '
    selection_go: #no better idea for the name :)
        CommandLine|contains:
            # "cmd" can be replaced by "powershell", "pwsh" or any other console like software
            - 'cmd'
            - 'pwsh' # (For PowerShell 7)
            - 'powershell' # (For PowerShell 5)
    selection_other:
        CommandLine|contains:
            - 'psexec'
            - 'paexec'
            - 'accepteula'
            - 'cmd /c '
            - 'cmd /k '
    condition: all of selection_*
```

Reply:
The order after the `-s` or `-i` is important because it's a "strong" indicator of PsExec/PAExec. If we split them into contains, we can have them spread out into the command line and it could generate false positives.

For example, if you say `cmd` contains like in the `selection_go` then it could match with the `cmd /c` and the `/s` can be somewhere in the CLI which is an FP in this case.